packit_service/worker/handlers/copr.py

```diff
@@ -479,3 +496,157 @@ def handle_testing_farm(self):
                         "build_id": self.build.id,
                     },
                 ).apply_async()
+
+    def handle_scan(self):
+        """
+        If configured, try to find a job that can provide the base SRPM,
+        download both SRPM and base SRPM and trigger the scan in OpenScanHub.
+        """
+        if (
+            self.db_project_event.type != ProjectEventModelType.pull_request
+            or self.build.target != "fedora-rawhide-x86_64"
+            or not self.job_config.osh_diff_scan_after_copr_build
```

Comment:
If we want to get this quickly and safely to stage/prod, what about starting with the opt-in first?

Reply:
(I mean like for a short while to check this on our packages and after a week or two, change the default...)

Reply:
yes we have been thinking about it, see https://github.com/packit/packit-service/pull/2472#issuecomment-2247519318, so I will probably change the default in the `packit` temporarily

Reply:
I have been already running mass scans successfully on production. If we keep this opt-in, the level of stress testing would be pretty limited. But I would leave it on the Packit leads to decide about it.

But I agree, that in case of problems, you should be able to quickly turn it off.

Reply:
@siteshwar ah ok, that might be a good point as well, we will discuss today

Reply:
added being able to control this via env var on our side